conf/test_full.config

```diff
@@ -10,19 +10,21 @@
 ----------------------------------------------------------------------------------------
 */
 
+cleanup = true
```

Comment:
Whats that about?

Reply:
On (successful) cleanup it automatically deletes the contents of the `work` folders to take up less space on s3